src/components/greeting.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { useSelector, useDispatch } from 'react-redux';
```

Comment:
- [ ] Please kindly ensure that eslint is not disabled as against the general requirement, this specific rule usually warns or throws an error when your code imports a module or dependency that is considered "extraneous" or unnecessary, often from the perspective of package.json's dependencies, devDependencies, or peerDependencies. Perhaps try and delete & reinstall node_modules, it may resolve it. ℹ️ 

**Kindly take note of this for the three other files where eslint is disabled as well.**

Reply:
Hi @OyePriscilla! Yes, you are right but I did some research but could not find any way to fix it.  The error is saying that you have to add react-redux to the developers in package.json but I checked that it is already there. I don't know where this error is coming from. If you have time please give me a solution.